test/controllers/DashboardControllerSpec.scala

```diff
@@ -19,10 +19,9 @@ package controllers
 import base.SpecBase
 import controllers.actions.TestAuthRetrievals.Ops
 import generators.ModelGenerators
-import models.UserAnswers
+import models._
```

Comment:
There is no need to use a wildcard import. Import only what is necessary:
```suggestion
import models.{Due, Incomplete, Overdue, Received, UserAnswers}
```